orchestra/analytics/latency.py

```diff
@@ -51,6 +53,8 @@ def work_time_df(projects, human_only=True, complete_tasks_only=True):
             human_only, complete_tasks_only)
         for project in projects))
     df = DataFrame(time_data)
+    df['start_datetime'] = df['start_datetime'].astype('datetime64[ns]')
+    df['end_datetime'] = df.start_datetime + df.calendar_time
```

Comment:
L128 seems to use end_datetime during the construction of the dataframe—why not add this column there?